commands.py

```diff
@@ -93,11 +107,35 @@ def run(self, _: sublime.Edit) -> None:
         debounced(
             functools.partial(plugin.request_get_completions, self.view),
             FEATURES_TIMEOUT,
-            lambda: not get_copilot_view_setting(self.view, "is_waiting", False),
+            lambda: not get_copilot_view_setting(self.view, "is_waiting_completions", False),
```

Comment:
I think we should hide this logic in the plugin (e.g. `plugin.is_waiting_completions()`)